src/core/drive/form_submission.ts

```diff
@@ -136,6 +136,8 @@ export class FormSubmission {
 
   requestSucceededWithResponse(request: FetchRequest, response: FetchResponse) {
     if (response.clientError || response.serverError) {
+      this.result = { success: false, fetchResponse: response }
+      dispatch("turbo:submit-failed", { target: this.formElement, detail: { formSubmission: this, ...this.result } })
       this.delegate.formSubmissionFailedWithResponse(this, response)
```

Comment:
I am actually not confident this code is ever called. If the fetchResponse is not a 2xx it will not be `ok` and therefore only the `requestFailedWithResponse` method will be called.

I added it here for safety, but this branch of code might not ever be called.